src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets

```diff
@@ -52,4 +52,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(NuGetBuildTasksPackTargets)"
           Condition="Exists('$(NuGetBuildTasksPackTargets)') AND '$(ImportNuGetBuildTasksPackTargetsFromSdk)' == 'true'"/>
+
+  <!-- Import targets from Microdoft.NET.Build.Container.targets -->
+  <PropertyGroup>
+    <_ContainersTargetsDir Condition=" '$(_ContainersTargetsDir)'=='' ">$(MSBuildThisFileDirectory)..\..\..\Containers\build\</_ContainersTargetsDir>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props"
+          Condition="'$(EnableSdkContainerSupport)' == 'true' AND Exists('$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props')" />
+
+  <Import Project="$(_ContainersTargetsDir)Microsoft.NET.Build.Containers.targets"
+    Condition="'$(EnableSdkContainerSupport)' == 'true'
+             AND Exists('$(_ContainersTargetsDir)Microsoft.NET.Build.Containers.targets')
+             AND '$(TargetFramework)' != ''" />
```

Comment:
I think we need to remove `EnableSdkContainerSupport` from these conditions, and instead put that condition on the `PublishContainer` target directly - this will enable someone to call `-t:PublishContainer` at the solution level and have the target be available (but not enabled) for all projects.

Reply:
Okay, I wanted to avoid importing these files when publishing is not required.

Reply:
Here's an example of a scenario that should become possible after we move the Container targets into the base SDK:

* myapp.sln
  * myapp.lib.csproj
  * myapp.web.csproj (references myapp.lib)
  * myapp.cli.csproj (references myapp.lib, has manually set `EnableSdkContainerSupport` to `true`)
  * myapp.tests.csproj (references myapp.lib)

We should be able to run `dotnet publish -t:PublishContainer` on `myapp.sln` and get containers generated for `myapp.web` and `myapp.cli` as a result.